### ISSUE TITLE
Marker Path not woorking with the Builder

### ISSUE THREAD

**Issue by StrangeW** [BuildCraft/BuildCraft#3983]
Marker Path not woorking with the Builder.

If I connect a constructor with a Marker Path, the constructor does not work properly after feeding it.

Another thing, an updated Wiki with the version number would be nice.

Thinks

**BuildCraft** version: 7.99.14
**Forge** version: 14.23.1.2555
Link to crash report or log: Not crach
Singleplayer or multiplayer: Singleplayer 
Additional information:

**Comment by AlexIIL** [comment]
Yeah, support hasn't been re-added yet. I'll leave this open as a reminder that this still needs to happen. 

We're working on both an in-game wiki (The BuildCraft Guide Book), and we're hoping that it can be used online as a wiki as well.

**Comment by StrangeW** [comment]
After several tests, it seems that the path ahead before construction is to take place.
The last location being built normally

**Comment by AlexIIL** [comment]
Sorry, I should have actually looked at it before replying.

It seems as though there are two problems:

 - [ ] It will immediately skip one section without finishing building
 - [ ] It only moves forward by a single block, rather than the size of the previous blueprint (so they need to move along the path by as little as possible, without overlapping with the previously built section).

**Comment by StrangeW** [comment]
It displays the build frame from beginning to end.

But he does not build and eventually build that last of the path

He also tries to destroy the blocks on the path, but goes on as a result and leaves the breaking particles in progress.

**Comment by AlexIIL** [comment]
In my dev environment loading the builder crashed, so I'd recommend removing it before it might do that to you.

Anyway, thanks for the report! This will be looked into sometime in the future.

**Comment by StrangeW** [comment]
Thank you
